### PR TITLE
HLM-5926 UI audit changes and enabling upload of multi sheet data

### PR DESCRIPTION
**Pull request by siddhant-nawale-egov**

### CODE REVIEW

File: micro-ui/web/micro-ui-internals/packages/modules/hcm-microplanning/src/components/MicroplanPreview.js [hunk 1]
```diff
@@ -141,7 +141,7 @@ const MicroplanPreview = ({
   // check if data has changed or not
   const updateData = useCallback(() => {
     if (!dataToShow || !setMicroplanData) return;
-    setMicroplanData((previous) => ({ ...previous, microplanPreview: dataToShow }));
+    setMicroplanData((previous) => ({ ...previous, microplanPreview: {previewData:dataToShow} }));
```
[the remark this answers is not in the document]
Comment (siddhant-nawale-egov) [reply micro-ui/web/micro-ui-internals/packages/modules/hcm-microplanning/src/components/MicroplanPreview.js:144]: the data i want to store in microplanData is a state variable so it is need to be this way.

File: micro-ui/web/micro-ui-internals/packages/modules/hcm-microplanning/src/components/Mapping.js [hunk 2]
```diff
@@ -674,7 +672,7 @@ const BoundarySelection = memo(
                 width: "100%",
                 padding: "0 0 1rem 1.3rem",
               }}
-              popupModuleMianStyles={{ padding: 0, margin: 0, maxWidth: "31.188rem" }}
+              popupModuleMianStyles={{ padding: 0, margin: 0, width: "31.188rem" }}
```
[the remark this answers is not in the document]
Comment (siddhant-nawale-egov) [reply micro-ui/web/micro-ui-internals/packages/modules/hcm-microplanning/src/components/Mapping.js:675]: its a external library component and doesnt have any className field so this cannot be done

File: micro-ui/web/micro-ui-internals/packages/modules/hcm-microplanning/src/components/MicroplanPreview.js [hunk 3]
```diff
@@ -357,7 +357,7 @@ const MicroplanPreview = ({
           </Modal>
         )}
         {toast && toast.state === "error" && (
-          <Toast style={{ bottom: "5.5rem", zIndex: "9999999" }} label={toast.message} isDleteBtn onClose={() => setToast(null)} error />
+          <Toast style={{ zIndex: "9999999" }} label={toast.message} isDleteBtn onClose={() => setToast(null)} error />
```
[the remark this answers is not in the document]
Comment (siddhant-nawale-egov) [reply micro-ui/web/micro-ui-internals/packages/modules/hcm-microplanning/src/components/MicroplanPreview.js:360]: that is the prop name, there is no mistake

File: micro-ui/web/micro-ui-internals/packages/modules/hcm-microplanning/src/components/MicroplanPreview.js [hunk 4]
```diff
@@ -339,7 +339,7 @@ const MicroplanPreview = ({
               width: "100%",
               padding: "1rem",
             }}
-            popupModuleMianStyles={{ padding: 0, margin: 0, maxWidth: "31.188rem" }}
+            popupModuleMianStyles={{ padding: 0, margin: 0, width: "31.188rem" }}
```
[the remark this answers is not in the document]
Comment (siddhant-nawale-egov) [reply micro-ui/web/micro-ui-internals/packages/modules/hcm-microplanning/src/components/MicroplanPreview.js:342]: It is that way in the model component from library

File: micro-ui/web/micro-ui-internals/packages/css/src/components/microplanning.scss [hunk 5]
```diff
@@ -666,16 +646,16 @@ $border-color: rgba(214, 213, 212, 1);
 
         .excel-wrapper {
             width: 100%;
-            padding: 1rem 0.5rem;
+            padding: 1rem 0.5rem 0 0.5rem;
             display: flex;
             flex-direction: column;
-            gap: 1.25rem;
+            gap: 0.5rem;
```
[the remark this answers is not in the document]
Comment (siddhant-nawale-egov) [reply micro-ui/web/micro-ui-internals/packages/css/src/components/microplanning.scss:652]: It gets overridden if !important is not used

File: micro-ui/web/micro-ui-internals/packages/css/src/components/microplanning.scss [hunk 6]
```diff
@@ -451,6 +483,11 @@ $border-color: rgba(214, 213, 212, 1);
             .deletebutton {
                 background-color: rgb(255, 255, 255, 0);
                 border: none;
+                display: flex !important;
+                flex-direction: row;
+                align-items: center !important;
+                justify-content: center;
+                margin: 0 1rem;
```
[the remark this answers is not in the document]
Comment (siddhant-nawale-egov) [reply micro-ui/web/micro-ui-internals/packages/css/src/components/microplanning.scss:490]: It is getting overriden

File: micro-ui/web/micro-ui-internals/packages/css/src/components/microplanning.scss [hunk 7]
```diff
@@ -27,6 +26,37 @@ $border-color: rgba(214, 213, 212, 1);
...
+            margin: 0 0 0 1rem !important;
+        }
+    }
+
+    .custom-button-left-icon {
+        .icon {
+            margin: 0 1rem 0 0 !important;
+        }
+    }
+}
```
[the remark this answers is not in the document]
Comment (siddhant-nawale-egov) [reply micro-ui/web/micro-ui-internals/packages/css/src/components/microplanning.scss:58]: It gets overriden

**Review by nipunarora-eGov — APPROVED** [review]

**Pull request merged**